Jenkinsfile

```diff
@@ -21,21 +21,44 @@ pipeline {
                 sh 'sbt Compile/compile Test/compile IntegrationTest/compile'
             }
         }
-        //stage('Test') {
-        //    steps {
-        //        sh 'sbt test'
-        //    }
-        //}
-        //stage('Integration test') {
-        //    steps {
-        //        sh 'sbt IntegrationTest/test'
-        //    }
-        //}
+        stage('Test') {
+            steps {
+                sh 'sbt "set ThisBuild/coverageEnabled := true" test'
+            }
+        }
+        stage('Integration test') {
+            steps {
+                sh 'sbt "set ThisBuild/coverageEnabled := true" IntegrationTest/test'
+            }
+        }
     }
     post {
-        //always {
-        //    junit '**/target/test-reports/*'
-        //}
+        always {
+            junit allowEmptyResults: true, testResults: '**/target/test-reports/*'
```

Comment:
Is the purpose of setting allowEmptyResults so that we can create unit tests without an output or maybe create placeholder unit tests? I'm wondering if we'd ever have an issue where an error within the unit test would result in an empty result.

Reply:
It's to handle the current case where there are no tests defined / if the format-check or compile stages fail and tests never run. Without "allowEmptyResults" another error will show up in the logs, which can be a red herring to the actual problem.